build/buildlib.spec

```diff
@@ -47,6 +47,15 @@ central,org.apache.xmlgraphics,xmlgraphics-commons,2.1,jar,build/lib/fop,B61132D
 central,org.apache.commons,commons-io,1.3.2,jar,build/lib/fop,B6DDE38349BA9BB5E6EA6320531EAE969985DAE5
 
 
+#graphql
+central,javax.servlet,javax.servlet-api,3.0.1,jar,build/lib/graphql,6BF0EBB7EFD993E222FC1112377B5E92A13B38DD
+central,com.graphql-java,graphql-java,13.0,jar,build/lib/graphql,F3D5C387CF09A13922F719FC984E7B6CBF2A3CCF
+central,com.graphql-java-kickstart,graphql-java-servlet,8.0.0,jar,build/lib/graphql,F0FEE2A1DD3715EE93473A54FA93F7CFDFC24E46
+central,com.graphql-java,graphql-java-extended-scalars,1.0,jar,build/lib/graphql,D798F4D543331B26E5E0B359D84D775F66B0E3E0
+central,com.google.guava,guava,24.1.1-jre,jar,build/lib/graphql,2E3014320A8005E3F3C1800CB246ED42DB8CAB81
```

Comment:
can this be moved to test?

Reply:
there are hard dependencies on these products in the code. I cannot move these into the test.
I can replace guava with handwritten code, but not the other three.

Reply:
that's not a problem.